### PR TITLE
Add support for corner radius modifier

### PR DESCRIPTION
**Pull request by kerrmarin**
Adds support for cornerRadius property via the --radius modifier in sketch layers

### CODE REVIEW

**Comment by kerrmarin** [comment]
@cdemetriadis do you want to put a quick summary of what this means for a sketch document here?

File: lib/processSketch.js [hunk 1]
```diff
@@ -53,4 +54,20 @@ const sortByName = (a, b) => {
 	return a1> b1? 1: -1;
 }
 
+const extractModifiers = (sketch) => {
+    const modifiers = sketch.getPages()[0].layers.filter((layer) => {
+        return layer.name.includes("--")
```
Comment (atomoil) [inline lib/processSketch.js:59]: Could you add a comment explaining what the `--` logic is for?

Comment (cdemetriadis) [reply lib/processSketch.js:59]: Modifiers are used for a very specific use case. In fact, we only use them for one reason, and that's to add corner radius to selected layers.

Sketch has chosen to not include Corner Radius into the Layer Style, and this led us to the modifiers. Much like adding an Atom in the theme, a designer can also add a modifier which sets the corner radius.

Example:
If we need to add rounded corners to the `/*/*/*/background` atom, we need to create a new atom named `/*/*/*/background --radius` and include a layer with the rounded corner settings. 

The above also adheres to the overarching concept. You can have a default setting for the Radius `/*/*/*/background --radius` modifier, or you can also target more specific atoms eg. `home/footer/primaryButton[selected]/background --radius`.

**Comment by atomoil** [comment]
Are there any other modifiers it’s at all conceivable anyone (even a madman) would want to use?

The identifiers you’ve mentioned start with `/` - should they? Or should any initial `/` be stripped out?

Are you planning to add the explanation to the README and an abbreviated form next to the code?

**Comment by cdemetriadis** [comment]
Modifiers can be used (in the future) for anything that is not saved in the Layer or Text Styles. 

Currently, we're only introducing modifiers for Radius, but we could also have modifiers for Resizing options like **Pin to Edge** and **Size Fix** etc.

The initial `/` can be stripped. We know that the Text & Layer Styles are `*/*/*/atomName`.

Once we finalise the Modifier functionality, I will be updating the README to include it.

**Comment by kerrmarin** [comment]
@atomoil this is ready for review again

File: lib/processSketch.js [hunk 2]
```diff
@@ -53,4 +54,38 @@ const sortByName = (a, b) => {
...
+
+Example:
+If we need to add rounded corners to the  * / * / * / background atom,
+we need to create a new atom named * / * / * / background --radius and include
+a layer with the rounded corner settings.
+
+The above also adheres to the overarching concept. You can have a default
+setting for the Radius * / * / * / background --radius modifier, or you
+can also target more specific atoms eg. home/footer/primaryButton[selected]/background --radius.
+*/
```
Comment (atomoil) [inline lib/processSketch.js:73]: Can this get added to a README.md?

**Pull request merged**